test/test_transforms_v2_refactored.py

```diff
@@ -295,9 +295,9 @@ def check_transform(transform_cls, input, *args, **kwargs):
     _check_transform_v1_compatibility(transform, input)
 
 
-def transform_cls_to_functional(transform_cls):
+def transform_cls_to_functional(transform_cls, **default_kwargs):
```

Comment:
With this, we can set attributes that are available on the transform, but not on the functional, e.g. `p=1` for randomly applied transforms.